### ISSUE TITLE
[ratio] Cvar to disable guards queue

### ISSUE THREAD

**Issue by shanapu** [shanapu/MyJailbreak#287]
https://forums.alliedmods.net/showthread.php?t=306088

**Comment by Javierko** [comment]
Or maybe you can make another plugin to ratio without queue and put in into modules file.

**Comment by Javierko** [comment]
After some testing on server, sometimes happends nobody can join to guards, it happends only on map start first round, after that "first" round after changing map, it works. No errors.

**Comment by shanapu** [comment]
Is it possible you have `sm_ratio_force_t "1" - force player on connect to join T side` and these players who can't connect are no admins/vips or you have additionally `sm_ratio_vip_bypass 0`?

**Comment by Javierko** [comment]
Yes, i have allowed sm_ratio_force_t 1 and i have allowed too sm_ratio_vip_bypass 1, i will try to disable vip bypass.
